pkgs/desktops/gnome/extensions/extensionRenames.nix

```diff
@@ -42,6 +41,7 @@
   "true-color-window-invert@lynet101" = "true-color-window-invert";
   "volume_scroller@francislavoie.github.io" = "volume-scroller-2";
   "openweather-extension@penguin-teal.github.io" = "openweather-refined";
+  "fuzzy-clock@keepawayfromfire.co.uk" = "fuzzy-clock-2";
```

Comment:
This one does not need to be preserved. The renames are here for when previously colliding package resolves the collision by changing its `pname` on e.g.o.

In this case, the extension got dropped from `gnomeExtensions` since it only supports GNOME ≤ 45 so the rename does not do anything.